src/widgets/refinement-list/defaultTemplates.js

```diff
@@ -1,7 +1,7 @@
 export default {
   header: '',
   item: `<label class="{{cssClasses.label}}">
-  <input type="checkbox" class="{{cssClasses.checkbox}}" value="{{name}}" {{#isRefined}}checked{{/isRefined}} />{{name}}
+  <input type="checkbox" class="{{cssClasses.checkbox}}" value="{{name}}" {{#isRefined}}checked{{/isRefined}} />{{{highlighted}}}
```

Comment:
Well after thinking about that, I don't think it's an issue. If users customized the template and they are using `name` it will still work. If they are using search for facet value, the value of `highlighted` will be set to name. I think we're safe actually.